ladybug_comfort/_loadmannequin.py

```diff
@@ -18,6 +18,6 @@ def load_solarcal_splines():
             'standing': csv_to_num_matrix(cur_dir + '/_mannequin/standingspline.csv')}
     except IOError:
         solarcal_splines = {}
-        print ('Failed to import projection factor splines from CSV.'
-               '\nA simpler interoplation method for Solarcal will be used.')
+        print('Failed to import projection factor splines from CSV.'
```

Comment:
Shouldn't this raise an error? won't this break something down the line? Also why don't you hard-code the values here instead of using a csv file? Is it something that user might need to change?

Reply:
1. No, there is an [alternative method to approximate the values in the table using interpolation, which is endorsed by ASHRAE-55](https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/solarcal.py#L556-L641). It defaults to this when it fails to load the splines, which are more precise but, as I said above, they are not always necessary.
2. It's large and isn't readable in a .py format (you really need Excel or Google Sheets to make any sense of it). Look at [the two CSVs](https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/_mannequin/seatedspline.csv) and, if you still feel like it should be hard-coded, then let me know.

Reply:
Yes. Those files are something!